index.js

```diff
@@ -0,0 +1,20 @@
+import express from 'express';
+import cors from 'cors';
+import { adminRouter } from './Routes/Admin.js';
+
+const app = express();
+
+const corsOptions = {
+  origin: "http://localhost:3001",
+ /* origin: "http://localhost:5173",*/
```

Comment:
With the current implementation of the project, and how you keep changing ports. it would be good that you just reference all origins..

`origin: *`